tests/assets/mnist.ipyg

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
Do not duplicate the file if not necessary, you can use the example file for integration tests. (Although not for unit tests, so you were right in your way)

Reply:
Done!